rfcs/0180-broken-package-removal.md

```diff
@@ -0,0 +1,94 @@
+---
+feature: broken-package-removal
+start-date: 2024-07-14
+author: Jörg "Mic92" Thalheim
+co-authors:
+shepherd-team: @emilazy @preisi @jopejoe1
+shepherd-leader: @jopejoe1
```

Comment:
Starting a YAML value with an `@` is invalid: https://yaml.org/spec/1.2.2/#example-invalid-use-of-reserved-indicators

Reply:
Is quoting ok?

Reply:
yes, absolutely

(an aside: I notice that something like half of the RFCs in this repo have YAML headers that are invalid in this way already...)

Reply:
good point i constantly notice the little github error

Reply:
Fixed.